NEWS.md

```diff
@@ -19,6 +20,70 @@ The format is based on [Keep a Changelog]
 | 2021-04-15 | [0.2.0]       | 0.13.0          | 5011          |
 | 2021-03-05 | [0.1.0]       | 0.12.0          | 4627          |
 
+## 0.6.1 - 2023-01-27
```

Comment:
Nit: The date doesn't match with the list above

Reply:
Fixed.